packages/row/src/css/index.js

```diff
@@ -25,6 +31,9 @@ export default {
       borderTop: 'none'
     }
   },
+  [`.psds-row.psds-theme--${themeNames.light}`]: {
+    borderTop: `1px solid ${colorsBorder.lowOnLight}`
```

Comment:
this border is super hard to see

Reply:
double checking...  seems to be set as that in figma.  it is subtle.